packages/react-layouts/package.json

```diff
@@ -28,7 +28,8 @@
     "@emotion/css": "^10.0.4",
     "@emotion/styled": "^10.0.7",
     "@quid/react-core": "^1.29.0",
-    "@quid/theme": "^1.29.0",
-    "react-router-dom": "^4.3.1"
+    "@quid/theme": "^1.30.0",
+    "react-router-dom": "^4.3.1",
```

Comment:
would it make sense to make this a peer dependency to give more flexibility to the consumer?

Reply:
Absolutely.

Reply:
We should move this to `devDependencies` now that it lives in `peerDependencies`. This way we get it while developing the library, but the consumers have to install it because it's a peerDependency